package.json

```
@@ -1,14 +1,31 @@
 {
   "name": "read",
   "version": "2.1.0",
-  "main": "lib/read.js",
+  "exports": {
+    "./package.json": "./package.json"
+  },
+  "type": "module",
+  "files": [
+    "src"
+  ],
+  "tshy": {
+    "exports": {
+      "./package.json": "./package.json"
```

Need to define a main.  Add `".": "./src/read.ts"` here.
```suggestion
      "./package.json": "./package.json",
      ".": "./src/read.ts"
```